elements/lisk-transactions/src/15_proof_of_misbehavior_transaction.ts

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { getAddressFromPublicKey } from '@liskhq/lisk-cryptography';
+import { validator } from '@liskhq/lisk-validator';
+
+import {
+	BaseTransaction,
+	StateStore,
+	StateStorePrepare,
+} from './base_transaction';
+import { convertToAssetError, TransactionError } from './errors';
+import { BlockHeader, TransactionJSON } from './transaction_types';
+import {
+	getBlockBytes,
+	getBlockBytesWithSignature,
+	isPunished,
+	validateSignature,
+} from './utils';
+
+const proofOfMisbehaviorAssetFormatSchema = {
+	type: 'object',
+	required: ['header1', 'header2'],
+	properties: {
+		header1: {
```

Comment:
header2 is missing?

Reply:
Also I suspect both header will have same schema so maybe define a separated schema for header and reference from this one?